.circleci/config.yml

```diff
@@ -49,8 +55,14 @@ jobs:
       - run:
           name: Build
           command: |
+            source `find /opt/ros -name setup.bash | sort | head -1`
```

Comment:
Why add the extra source? Shouldn't the source in "Set Up Container" be enough?

Reply:
It runs each `- run` step as an independent step in the container and, apparently, does it with a non-interactive terminal. This means that it doesn't source `~/.bashrc` in the container nor run the `CMD` defined in the Dockerfile so, unfortunately, you have to do this for it to find packages in the install.

Reply:
Should this line also be added to the kinetic Build step as well then? I'm confused why there are difference between the kinetic and melodic steps. The kinetic Lint step has a source but the melodic Lint step doesn't.

Reply:
I need to do more testing with and without it. Do you mind if we move that to another PR?